sanitizer/search_fallback.js

```diff
@@ -19,7 +19,6 @@ module.exports.middleware = function( req, res, next ){
   // log the query that caused a fallback since libpostal+new-queries didn't return anything
   if (req.path === '/v1/search') {
     var queryText = logging.isDNT(req) ? '[text removed]' : req.clean.text;
-    logger.info(queryText);
```

Comment:
Weird, I don't recall removing that, but I guess it doesn't the code doesn't do much without it.